models/bamf_nnunet_mr_liver/utils/BamfProcessorRunner.py

```diff
@@ -0,0 +1,66 @@
+"""
```

Comment:
_**INFO**: Please copy all changes over to the other PR's where you use your custom `BamfProcessorRunner` Module._